app/mailers/jobapps_mailer.rb

```diff
@@ -42,6 +42,16 @@ def interview_reschedule(interview)
          reply_to: reply_to
   end
 
+  def send_note_for_later(application_record)
```

Comment:
It's an attribute of record. The line /@record = application_record allows me to use any attributes or methods of that application record in the mailer view, which you will see when you get to that portion.